karapace/schema_models.py

```diff
@@ -188,21 +229,41 @@ def parse(
     references: Sequence[Reference] | None = None,
     dependencies: Mapping[str, Dependency] | None = None,
     normalize: bool = False,
+    dependencies_compat: bool = False,
```

Comment:
I am guessing this is some flag you want to introduce so we can opt-in/out of the referencing logic, as it is it is hard coded, so maybe you can add it to the config.

Reply:
fixed